lms/djangoapps/commerce/tests/test_utils.py

```diff
@@ -269,6 +269,37 @@ def test_ecommerce_refund_failed_process_notification_sent(self, mock_send_notif
         assert call_args[0] == (course_entitlement.user, [1])
         assert refund_success
 
+    @httpretty.activate
+    @patch('lms.djangoapps.commerce.utils.send_notification_email_to_support', return_value=True)
+    def test_ecommerce_refund_failed_process_notification_email_sent(self, mock_send_notification):
+        """
+        Test that email is sent for notification instead of creating a ticket on zendesk when
+        `ENABLE_EMAIL_INSTEAD_ZENDESK` feature flag is set to `True`
```

Comment:
Kindly have a look at the docstrings again. I think these are a little ambiguous.

Reply:
Do you have something in mind? I can't think of something simpler.